server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_1__alter_notification_schema.sql

```diff
@@ -0,0 +1,104 @@
+DROP TABLE IF EXISTS `notification_channel`;
+CREATE TABLE `notification_channel` (
```

Comment:
better use `CREATE TABLE IF NOT EXISTS notification_channel`

Reply:
Sorry for missing this. Added.